nservicebus/upgrades/all-versions.include.md

```diff
@@ -0,0 +1,2095 @@
+### NServiceBus
+
+<table>
+  <tr>
+    <th>Version</th>
+    <th>Released</th>
+    <th>Supported until</th>
```

Comment:
Did u consider making this a markdown table. It would make the file much smaller and concise.